### DISCUSSION TITLE
Regarding release of stable version 15.0.0

### DISCUSSION THREAD

**Discussion by akhilaet-92** [stephanrauh/ngx-extended-pdf-viewer#1521] [Q&A]
Hi @stephanrauh, Just wanted to know when we can expect the stable version 15.0.0 published? Please refer the below link for the related issue.

https://github.com/stephanrauh/ngx-extended-pdf-viewer/issues/1508

**Comment by stephanrauh** [comment]
Soon. I want to publish it during the next seven day. Unfortunately, my job is currently a bit demanding, so I don't have as much leisure time as I need to polish and finish the version. However, you can help me by testing the version. I've found a couple of bugs related to forms, but maybe there are more bugs lurking around?

**Reply by akhilaet-92** [reply]
Thanks for the update.

**Reply by stephanrauh** [reply]
I've just solved #1516. I considered it a blocker. I'm going to run another exhaustive test before publishing the new stable version, but now I'm positive I can publish it this week-end.

**Reply by akhilaet-92** [reply]
Version 15.0.0 is working fine. Thank you for the support.
